src/TTrigger.cpp

```diff
@@ -1497,3 +1497,19 @@ void TTrigger::decodeColorPatternText(const QString& patternText, int& fgColorCo
         bgColorCode = scmIgnored;
     }
 }
+
+QString TTrigger::packageName(TTrigger* pTrigger) {
```

Comment:
Remember that this will only work for Trigger items - you'll need similar methods for each of the other Mudlet item types...